lib/core/viewer/components/viewer.component.html

```diff
@@ -212,6 +212,7 @@ <h2>{{ 'ADF_VIEWER.LOADING' | translate }}</h2>
                                             [blobFile]="blobFile"
                                             [urlFile]="urlFileContent"
                                             [nameFile]="displayName"
+                                            [cacheType]="cacheTypeForContent"
```

Comment:
do we need a new input? or we can add it always?

Reply:
Due the fact that this affects the caching for when we get the file it might be better to have a flexible strategy. Maybe the no-cache is something that makes sense for us but not in some other cases when the viewer is used.